Source/Examples/Avalonia/ExampleBrowser/MainWindow.axaml

```diff
@@ -11,6 +13,10 @@
     <Window.DataContext>
       <exampleBrowser:MainViewModel />
     </Window.DataContext>
+
+    <Window.Resources>
+        <exampleBrowser:NotNullBooleanConverter x:Key="NotNullBooleanConverter" />
```

Comment:
Avalonia ships with one of these: https://docs.avaloniaui.net/docs/reference/built-in-data-binding-converters so shouldn't need our own or the resource definition
Can use `Converter={x:Static ObjectConverters.IsNotNull}` to reference it
(can also use `!!` notation, but that's cryptic)